content/slides/intro-to-jupyterlab.qmd

```diff
@@ -105,6 +113,41 @@ pop-up in CryoCloud.
 
 ::::::
 
+## Running terminals and kernels ![](/_media/jupyterlab_running_terminals_kernels_icon.png){width="3%" style="vertical-align:middle"}
```

Comment:
I thought it might be handy to show this view of running terminals and kernels. E.g., if a notebook gets 'stuck', a user might want to use this interface to stop the kernel associated with it.

That said, we haven't introduced notebooks (or terminals) at this point yet either. Maybe this should be moved to later? Or should we just drop it?

Reply:
I like having this content, I don't think we should drop it. But I can see the benefit of moving it later.